test/local/features.js

```diff
@@ -33,9 +34,10 @@ describe('features', () => {
     'interface is correct',
     () => {
       assert.equal(typeof features, 'object', 'object type should be exported')
-      assert.equal(Object.keys(features).length, 2, 'object should have four properties')
+      assert.equal(Object.keys(features).length, 3, 'object should have four properties')
```

Comment:
Mismatch? 
`Object.keys(features).length, 3, 'object should have` **four** `properties'`